python/cugraph/cugraph/gnn/data_loading/bulk_sampler.py

```diff
@@ -196,54 +201,44 @@ def flush(self) -> None:
         else:
             sample_fn = cugraph.dask.uniform_neighbor_sample
             self.__sample_call_args["_multiple_clients"] = True
+            self.__sample_call_args[
+                "label_to_output_comm_rank"
+            ] = self.__get_label_to_output_comm_rank(min_batch_id, max_batch_id)
```

Comment:
I'm assuming `black` did this, but it seems like this formatting could be nicer :)